frontend/src/Command/Commands/version.ts

```
@@ -26,10 +30,7 @@ class VersionCommand extends BaseCommand {
     };
   }
 
-  async getSuggestions(
-    _input: string,
-    _parameters: Record<string, string>
-  ): Promise<SuggestionList> {
+  async getSuggestions({}: SuggestionsParams): Promise<SuggestionList> {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)